### DISCUSSION TITLE
Route with params/props (Not in URL)

### DISCUSSION THREAD

**Discussion by iampapagray** [hannoeru/vite-plugin-pages#399] [Q&A]
What structure should be used to generate a route that accepts a param(s).

Eg.

i have `pages/verify.vue` which produces a verify route.
However when i use 
````
router.push({ name: 'verify', params: {phone: '+18383'} })
````

the route is changed to the verify page but the params are discarded because they were not preset on the route.

What to do to solve this issue without without switching to manual route definitions?
**NOTE: I prefer not to have the param values in the url!**

**Comment by jbaubree** [comment]
I think this link will help you `https://github.com/vuejs/router/blob/main/packages/router/CHANGELOG.md#414-2022-08-22`
